packages/react-internal/src/components/pickers/BasePicker.tsx

```diff
@@ -274,15 +274,14 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends React.Componen
           input: css('ms-BasePicker-input', legacyStyles.pickerInput, inputProps && inputProps.className),
           screenReaderText: legacyStyles.screenReaderOnly,
         };
-
     return (
       <div ref={this.root} className={classNames.root} onKeyDown={this.onKeyDown} onBlur={this.onBlur}>
         <FocusZone
           componentRef={this.focusZone}
           direction={FocusZoneDirection.bidirectional}
           shouldEnterInnerZone={this._shouldFocusZoneEnterInnerZone}
-          role={'combobox'}
-          id={this._ariaMap.combobox}
+          role={canAddItems ? 'combobox' : undefined}
+          id={canAddItems ? this._ariaMap.combobox : undefined}
           aria-label={this.props['aria-label']}
           aria-expanded={!!this.state.suggestionsVisible}
           aria-owns={suggestionsAvailable || undefined}
```

Comment:
Just one note -- `aria-label`, `aria-expanded`, and `aria-owns` will all be invalid on the picker with the role removed, so they should likely have the conditional as well.

It's not a big user impact since I'm guessing screen readers will just ignore all three, but it could cause errors from automated a11y checkers.

Reply:
Good call. But I think we should keep `aria-label` since 
1. "This attribute can be used with any typical HTML element; it is not limited to elements that have an ARIA role assigned."
2. the control is still focusable in this state

Reply:
@xugao I think only the children within this combo are focusable, since it's still on the older combobox pattern. If the element itself is focusable, it would probably need to keep the role 😅.

`aria-label` (and other accessible name attributes) are also prohibited on generic roles in ARIA 1.2: https://w3c.github.io/aria/#generic